src/status_im/utils/ethereum/stateofus.cljs

```diff
@@ -4,11 +4,16 @@
             [status-im.utils.ethereum.core :as ethereum]
             [status-im.utils.ethereum.ens :as ens]))
 
+(defn is-valid-eth-name? [ens-name]
+  (and ens-name
+       (string/ends-with? ens-name ".eth")))
+
 (defn is-valid-name? [ens-name]
-  (string/ends-with? ens-name ".stateofus.eth"))
+  (and ens-name
+       (string/ends-with? ens-name ".stateofus.eth")))
 
 (defn addr [web3 registry ens-name cb]
-  {:pre [(is-valid-name? ens-name)]}
+  {:pre [(is-valid-eth-name? ens-name)]}
```

Comment:
Keep in mind that `:pre` assertions are removed in production builds since we have https://cljs.github.io/api/compiler-options/elide-asserts set to true

Reply:
also the resolver is for stateofus names so I'm not sure it will work with eth names

Reply:
nevermind you are probably right about stateofus namespace

Reply:
@yenda it works for eth names, no idea how